nav2_costmap_2d/plugins/costmap_filters/keepout_filter.cpp

```diff
@@ -100,7 +112,7 @@ void KeepoutFilter::filterInfoCallback(
   RCLCPP_INFO(
```

Comment:
You have your work for the keepout filters in this branch - can you remove them so that this is only the speed limits work? Its hard to review with these mixed together

Reply:
Still a bunch of keepout stuff that was merged in already, please get rid of them